integrations/pgvector/src/haystack_integrations/document_stores/pgvector/document_store.py

```diff
@@ -117,6 +119,7 @@ def __init__(
         :param hnsw_ef_search: The `ef_search` parameter to use at query time. Only used if search_strategy is set to
             `"hnsw"`. You can find more information about this parameter in the
             [pgvector documentation](https://github.com/pgvector/pgvector?tab=readme-ov-file#hnsw)
+        :param language: The language to use for the full-text/hybrid search.
```

Comment:
let's add some information on where to find the supported languages.
I haven't found much
https://stackoverflow.com/questions/39751892/get-full-list-of-full-text-search-configuration-languages

Reply:
In regard to this, I can't really find a web/documentation reference either. However when executing the query "SELECT cfgname FROM pg_ts_config;" I got the list - > 
cfgname
--
simple
danish
dutch
english
finnish
french
german
hungarian
italian
norwegian
portuguese
romanian
russian
spanish
swedish
turkish